src/main/java/com/exasol/releasedroid/usecases/UserInput.java

```diff
@@ -165,9 +199,18 @@ public Builder repositoryName(final String repositoryName) {
         public UserInput build() {
             validateMandatoryParameters();
             validateGoalAndBranch();
+            validateLocalPath();
             return new UserInput(this);
         }
 
+        private void validateLocalPath() {
+            if ((this.localPath != null) && ((this.goal == Goal.RELEASE) || (this.branch != null))) {
+                throw new IllegalArgumentException(ExaError.messageBuilder("E-RR-6")
```

Comment:
Please keep track on the `pom` file (until we have a better place for doing it) of the next error code to use, for all the kinds: `E`, `W` and `S`

Reply:
I have quite a lot of different tags around the project. Don't want to spam I'm pom with them. Maybe I'll add a separate file for them, but I'll do it next time I touch some exceptions. Don't want to waste time on it now